pkg/controller/backup_controller.go

```diff
@@ -993,7 +996,7 @@ func (c *backupController) waitVolumeSnapshotReadyToUse(ctx context.Context,
 // change DeletionPolicy to Retain before deleting VS, then change DeletionPolicy back to Delete.
 func (c *backupController) deleteVolumeSnapshot(volumeSnapshots []snapshotv1api.VolumeSnapshot,
 	volumeSnapshotContents []snapshotv1api.VolumeSnapshotContent,
-	logger logrus.FieldLogger) {
+	logger logrus.FieldLogger, timeout time.Duration) {
```

Comment:
```suggestion
	logger logrus.FieldLogger) {
```
backupController already have resourceTimeout within its structs. Why is there a need to pass this function parameter? backupController should've already been set with this value.